source/manuals/accessibility.html.md.erb

```diff
@@ -0,0 +1,115 @@
+
+### Form controls not associated with a label
+
+All form controls should have a label associated with them that describes what the form control should be used for. [Placeholder text](https://w3c.github.io/html/sec-forms.html#the-placeholder-attribute) is not an acceptable alternative.
+
+Labels should provide a short but clear explanation of a form control. Further detail can be provided using a separate element. This should also be associated with the form control by using an attribute such as aria-describedby.
+
+### Meaningless or duplicated link text
+
+The text of a link should describe what that link points to. It should not rely on the surrounding text for context. Screen reader users can navigate pages by element type, which means that link text such as ‘click here’ or ‘read more’ does not help a user understand where the link goes.
+
+### Poor colour contrast
+
+Users can have a range of sight issues, including colour blindness or situational disabilities such as screen glare. Text on a web page should always be clear and readable and background images behind text should generally be avoided.
+
+WCAG 2.1 level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text. Contrast checking tools such as [WebAIM’s contrast checker](https://webaim.org/resources/contrastchecker/) can provide a quick way to test if text is readable.
+
+### Poor keyboard navigation
+
+Keyboard users rely on focus states to know where they are on a page. This means that elements such as links and form controls must have a clear focus state set in CSS to indicate when those elements have keyboard focus.
+
+This issue relates back to the use of semantic HTML. The use of inappropriate elements can remove the ability for keyboard users to access controls on a page, making the service unusable.
+
+### Inflexibility
+
+Services should have a responsive layout in order to work on any screen size. In addition, consideration and testing should be included for users who apply customisations to their web browsing, either through web browser controls or specific tools. Examples of these customisations include increasing the font size or applying a custom stylesheet, such as a high contrast theme.
+
+### Complex interactive pages
+
+Pages that provide detailed interaction for the user must be built with accessibility in mind. If page elements are updated dynamically using JavaScript then attributes such as aria-controls and aria-live should be used to ensure that screen reader users are informed when page content changes.
+
+### Images without alt text
+
+Non sighted users cannot understand an image unless meaningful alt text (alternative text) is provided. When used it should concisely convey the intent of the image and not necessarily describe it fully. For example, the alt text "a hearty Christmas feast" may be more helpful in conveying the intent of an image than "a plate with turkey, roast potatoes, stuffing, gravy, sprouts and carrots".
```

Comment:
Should we mention that an image element with blank alt text is better than an image element with no alt attribute?

Reply:
Yes, good shout. Have added.